api/rpc_test.go

```diff
@@ -59,7 +59,7 @@ func TestRPCCallsUnderlyingNode(t *testing.T) {
 	nd, server := setupNodeWithAuthedRPC(t, signer, verifier)
 	url := nd.RPCServer.ListenAddr()
 
-	adminToken, err := perms.NewTokenWithPerms(signer, perms.AllPerms)
+	adminToken, err := perms.NewTokenWithPerms(signer, perms.AllPerms, time.Minute)
```

Comment:
Should be fine for the test.

Reply:
Can you also add test where token is valid for some time, then is invalid after expiry? I know it's time-based so it's tricky but would be nice to have that coverage.